cmake/modules/RootMacros.cmake

```diff
@@ -2072,3 +2072,1331 @@ function(generateManual name input output)
 
   install(FILES ${output} DESTINATION ${CMAKE_INSTALL_MANDIR}/man1)
 endfunction()
+
+#-------------------------------------------------------------------------------
+#
+#  Former RoottestMacros.cmake starts here
```

Comment:
I would have found it much clearer to just move the file as is ....

Reply:
For the commit history maybe, but my hope was that by bringing these files together, more code duplication will be apparent and inspire code improvements along the way in the future.

Reply:
I think that having the file side by side would have had the same effect ... and preserve commit history (which is too often the only source of documentation of those routines) ...